<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>661</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>661</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SD-Flasher for Linux(Author:kangear)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver:130620</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>531</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>621</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Image File to Fuse...</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>30</y>
      <width>592</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_path">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>27</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBut_Brose">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>SD Drive(s)</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>592</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="tWid_scan">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>100</height>
        </size>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>80</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>10</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Physical Disk#</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Label</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Available</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Result</string>
        </property>
        <property name="textAlignment">
         <set>AlignHCenter|AlignVCenter|AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>97</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBut_Scan">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>761</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Reports</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>592</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextEdit" name="textEdit_fush">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>100</height>
        </size>
       </property>
       <property name="autoFormatting">
        <set>QTextEdit::AutoNone</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>97</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pBut_Fuse">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Fuse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
